app/Models/User.php

```diff
@@ -37,23 +37,30 @@ public function user() {
 	public function preferredLocale() {
 		return $this->language;
 	}
-	
+
 	public function sendPasswordResetNotification($token) {
-		if ($this->password !== '' || !($this->user_type === Worker::class || $this->user_type === ArtistManager::class || $this->user_type === Band::class)) {
+		if ($this->password !== '' || !(
```

Comment:
we can probably turn this if into ! in_array($this0>user_type,[Worker::class, etc])